aws/authn_task.json

```diff
@@ -106,86 +102,74 @@
                     "valueFrom": "arn:aws:secretsmanager:us-east-1:{{account_id}}:secret:bento/{{project}}/{{environment}}:neo4j_password::"
                 },
                 {
-                    "name": "MYSQL_DB_HOST",
+                    "name": "MYSQL_HOST",
                     "valueFrom": "arn:aws:secretsmanager:us-east-1:{{account_id}}:secret:bento/{{project}}/{{environment}}:rds_host::"
                 },
                 {
-                    "name": "MYSQL_DB_USERNAME",
+                    "name": "MYSQL_USER",
                     "valueFrom": "arn:aws:secretsmanager:us-east-1:{{account_id}}:secret:bento/{{project}}/{{environment}}:rds_username::"
                 },
                 {
-                    "name": "MYSQL_DB_PASSWORD",
+                    "name": "MYSQL_PASSWORD",
                     "valueFrom": "arn:aws:secretsmanager:us-east-1:{{account_id}}:secret:bento/{{project}}/{{environment}}:rds_password::"
                 }
             ],
             "environment": [
                 {
-                    "name": "PROJECT",
-                    "value": "{{project}}"
+                    "name": "DATE",
+                    "value": "{{date}}"
                 },
                 {
                     "name": "VERSION",
                     "value": "{{image_tag}}"
                 },
-                {
-                    "name": "SESSION_SECRET",
-                    "value": "abcd256asghaaamnkloofghj"
-                },
-                {
-                    "name": "IDP",
-                    "value": "nih"
-                },
-                {
-                    "name": "PROJECT",
-                    "value": "{{project}}"
+                // {
+                //     "name": "COOKIE_SECRET",
+                //     "value": "abcd256asghaaamnkloofghj"
+                // },
+                // {
+                //     "name": "SESSION_TIMEOUT",
+                //     "value": ""
+                // },
+                // {
+                //     "name": "TOKEN_SECRET",
+                //     "value": ""
+                // },
```

Comment:
these commented lines should be removed - I don't think the comments are valid json and this will break the template